luasql-exasol-0.2.1-1.rockspec

```diff
@@ -31,7 +31,8 @@ dependencies = {
     "luasocket >= 3.1.0-1",
     "luasec >= 1.3.2-1",
     "luaossl >= 20220711-0",
-    "lua-cjson == 2.1.0", -- pinned to prevent "undefined symbol: lua_objlen" in 2.1.0.6 (https://github.com/mpx/lua-cjson/issues/56)
+    -- "lua-cjson == 2.1.0", -- pinned to prevent "undefined symbol: lua_objlen" in 2.1.0.6 (https://github.com/mpx/lua-cjson/issues/56)
```

Comment:
```suggestion
```
Is pinning still required?

Reply:
No, the problem seems to be fixed. Removed the info and the commented line.